csrc/executor.cpp

```diff
@@ -1906,14 +1906,10 @@ std::vector<at::Tensor> FusionExecutor::runFusion(
       " provided number of outputs does not match fusion output");
 
   // Bind fusion inputs
-  ExpressionEvaluator expr_eval;
-  const auto& inputs = fusion()->inputs();
-  for (const auto i : c10::irange(inputs.size())) {
-    expr_eval.bind(inputs[i], *args[i]);
-  }
+  auto expr_eval = executor_utils::bindInputs(args, fusion());
 
   if (isExpressionEvaluated(fusion())) {
-    outputs = evaluateFusionOutputs(args, outputs, expr_eval);
+    outputs = evaluateFusionOutputs(outputs, expr_eval);
```

Comment:
nitpick: So ExpressionEvaluator wouldn't insert outputs to `args` like the other code path running with codegen. But looks like we are not using `args` later in this function nor its callers. I'm fine with this, but can we have a short comment highlighting that?